### DISCUSSION TITLE
Getting started with building homebox

### DISCUSSION THREAD

**Discussion by BoronFinder** [sysadminsmedia/homebox#327] [Q&A]
I'm completely new to Go, and I'd like to see if I can contribute to this project.
Since my coding experience is largely limited to scripting languages like VBA and a little bit of JS so far, I struggle mightily when it comes to compiling in order to try out changes.
Can somebody lend me a hand here?

So far, I managed to download and install Go 1.23.3 on my Raspberry Pi 3B and run and build a Hello World program.
I git cloned the Homebox project and figured out that I would probably need to build the backend*, since this is where the Go files live. So I went to the backend folder and entered "go build", but got back "no Go files in /home/pi/go/homebox/homebox/backend".
What do I need to do to build the backend?

**Comment by tonyaellie** [comment]
Hi!
You can find information on how to setup the project on our docs at https://homebox.software/en/contribute/get-started.
